service.go

```diff
@@ -21,7 +23,9 @@ type Service interface {
 	ServiceName() string
 	TrackName() string
 	URLRegex(string) bool
+	SearchRegex(string) bool
```

Comment:
it is used in FindServiceAndSearch to find the service.

same as URLRegex to find the service for the !add command